Kernel/Graphics/VGACompatibleAdapter.cpp

```diff
@@ -37,7 +37,7 @@ UNMAP_AFTER_INIT void VGACompatibleAdapter::initialize_framebuffer_devices()
 }
 
 UNMAP_AFTER_INIT VGACompatibleAdapter::VGACompatibleAdapter(PCI::Address address)
-    : GraphicsDevice(address)
+    : GraphicsDevice()
```

Comment:
This explicit ctor call is not necessary (more follow, so I'm gonna ignore them)